app/contribute/page.tsx

```diff
@@ -49,17 +49,67 @@ export default function About() {
                 <strong>Optimism</strong> &mdash; We are optimistic about the
                 positive impacts of AI, both now and in the future. We simply
                 believe that the best way to build safer and more responsible AI
-                systems is to learn from the mistakes of the past. This site is
+                systems is to learn from the mistakes of the past. This website is
                 not &quot;anti-AI&quot; &mdash; not by a long shot.
               </li>
               <li>
                 <strong>Reliability</strong> &mdash; Our project relies on
                 well-researched, verifiable instances of harm. We do not accept
                 anecdotal evidence. It is crucial that our catalog of harms is
                 as accurate as possible. More information about our sourcing and
-                verification process is below.
+                verification process is in the next section.
               </li>
             </ol>
+            <h2>Sourcing New Incidents</h2>
+            <p>When finding new incidents to add to Badness.ai, please ensure they meet
+              the following standards designed to uphold the quality and veracity of the
+              catalog.</p>
+            <ul>
+              <li>
+                Badness.ai is a catalog of real-world harms &mdash; demonstrative or not.
+                As a result, &ldquo;jailbreaks&rdquo; and other examples of
+                system vulnerabilities are generally not in scope for the
+                catalog, unless they caused harm or indicate novel
+                potential for harm. For example, the many examples of what people
+                were able to get ChatGPT to say via the
+                infamous <a href="https://www.cnbc.com/2023/02/06/chatgpt-jailbreak-forces-it-to-break-its-own-rules.html">DAN jailbreak</a> are
+                not in scope; these examples would not realistically occur
+                in the wild since they require the user to intentionally jailbreak the
+                underlying model. However, an example where a
+                user <i>unexpectedly</i> jailbreaks the underlying model and
```

Comment:
Could we simplify this language a big? E.g., examples where users encounter harmful output without coaxing the model into doing so? ("unexpectedly jailbreaks" doesn't quite parse for me)

Reply:
What if I remove the example altogether? I think "users encountering harmful output without coaxing the model into doing so" is more of a description of most examples in the catalog rather than a specific set of examples, so  I incorporated that language into the justification for why jailbreaks are not in scope in my latest commit:

https://github.com/badnessdotai/catalog/pull/57/commits/21e6115b249eb6e6a357564788c6217ba6951c43